### ISSUE TITLE
[color-picker] Starts of with the wrong color

### ISSUE THREAD

**Issue by DmitrySharabin** [color-js/elements#34]
E.g., for

```html
<color-picker space="oklch" color="oklch(50% 50% 180)"></color-picker>
```

we have

<img width="1174" alt="SCR-20240515-ojxe-2" src="https://github.com/color-js/elements/assets/9166277/cf4f3830-6051-40d6-882b-f6d55c3f1cab">

The color channels are set correctly, but the swatch shows the value from the element's definition:

https://github.com/color-js/elements/blob/61be842e960d269a8eaef9e34797b689b626fd44/src/color-picker/color-picker.js#L21

**Comment by DmitrySharabin** [comment]
It also doesn't respect the provided color. E.g., for 
```html
<color-picker space="oklch" color="oklch(10% 70% 30)"></color-picker>
```
we get the same color as in the picture above.

**Comment by LeaVerou** [comment]
Fixed by @DmitrySharabin in caa77bb925a3348c46168692b2c9c0a5c915e666
